ci/deps/actions-38-minimum_versions.yaml

```diff
@@ -49,6 +49,7 @@ dependencies:
   - scipy=1.7.1
   - sqlalchemy=1.4.16
   - tabulate=0.8.9
+  - tinycss2=1.0.0
```

Comment:
Is this actively maintained? This looks more or less abandoned

Reply:
Are you looking at tinycss rather than tinycss2? The later has commits from a few weeks ago on the bleeding edge.

Reply:
I checked https://github.com/Kozea/tinycss2/commits/master and there are like 25 commits in the last 2 years

Reply:
What can I say. There's not much more activity needed than that. It's a simple parser that just produces an AST from some CSS, only a few 100 lines of code if you strip out the comments and doc strings. Doesn't do any of the other fancy processing of interpreting the CSS content. That's all we need here to pull out the key-value pairs being passed in.